docs/REST API/rest-api.md

```diff
@@ -6376,32 +6369,73 @@ None.
 }
 ```
 
-### Get Apple Business Manager (ABM)
+### List Apple Business Manager (ABM) tokens
 
 _Available in Fleet Premium_
 
-`GET /api/v1/fleet/abm`
+`GET /api/v1/fleet/abm_tokens`
 
 #### Parameters
 
 None.
 
 #### Example
 
-`GET /api/v1/fleet/abm`
+`GET /api/v1/fleet/abm_tokens`
 
 ##### Default response
 
 `Status: 200`
 
 ```json
-{
-  "apple_id": "apple@example.com",
-  "org_name": "Fleet Device Management",
-  "mdm_server_url": "https://example.com/mdm/apple/mdm",
-  "renew_date": "2023-11-29T00:00:00Z",
-  "default_team": ""
-}
+[
+  {
+    "id": 1,
+    "apple_id": "apple@example.com",
+    "org_name": "Fleet Device Management Inc.",
+    "mdm_server_url": "https://example.com/mdm/apple/mdm",
+    "renew_date": "2023-11-29T00:00:00Z",
+    "terms_expired": false,
+    "macos_team": "💻 Workstations",
+    "ios_team": "📱🏢 Company-owned iPhones",
+    "ipados_team": "🔳🏢 Company-owned iPads"
```

Comment:
```suggestion
    "macos_team": { "name": "💻 Workstations", "team_id" 1}
    "ios_team": { "name": "📱🏢 Company-owned iPhones", "team_id": 2}
    "ipados_team": { "name": "🔳🏢 Company-owned iPads", "team_id": 3}
```

Reply:
@marko-lisica sorry I missed asking this in standup, but should the response object have a subkey? e.g. ```json
{
    "abm_tokens": [...]
}
```

Reply:
@jahzielv I'll add it to both `GET /api/v1/fleet/abm_tokens` and `GET /api/v1/fleet/vpp_tokens` since that's what we do in other endpoints.